### PR TITLE
MULE-18105:Sockets Connector User Guide Needs Connection Information.

### PR DESCRIPTION
**Pull request by emariotti3**

### CODE REVIEW

File: modules/ROOT/pages/sockets/sockets-connector-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+		<sockets:udp-requester-connection host="localhost" port="8085" />
+	</sockets:request-config>
+
+	<flow name="socketSearchDatabaseAndReturnResult">
+		<sockets:listener doc:name="Listener" config-ref="Sockets_Listener_config" outputMimeType="application/json"/>
+		<db:select doc:name="Select" config-ref="Database_Config">
+			<db:sql>#["SELECT first_name, last_name from SYSTEM.MUSICIANS where id = :id"]</db:sql>
+			<db:input-parameters><![CDATA[#[{ 'id': payload.id }]]]></db:input-parameters>
+		</db:select>
+		<set-payload value="#[output application/json --- 'name': payload[0].FIRST_NAME as String ++ ' ' ++ payload[0].LAST_NAME as String]" doc:name="Set Payload" />
```
Comment (marianogonzalez) [inline modules/ROOT/pages/sockets/sockets-connector-examples.adoc]: delete this. Perform this transformation in the `<sockets:content>` tag below

File: modules/ROOT/pages/sockets/sockets-connector-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+	<file:config name="File_Config" doc:name="File Config" >
+		<file:connection />
+	</file:config>
+	<flow name="socket-testFlow">
+		<sockets:listener doc:name="Listener"  config-ref="Sockets_Listener_config" outputMimeType="application/json" />
+		<db:select doc:name="Select"  config-ref="Database_Config">
+			<db:sql >#[&quot;SELECT first_name, last_name from SYSTEM.MUSICIANS where id = :id&quot;]</db:sql>
+			<db:input-parameters><![CDATA[#[{ 'id': payload.id }]]]></db:input-parameters>
+		</db:select>
+		<set-payload value="#[output application/json --- 'name': payload[0].FIRST_NAME as String ++ ' ' ++ payload[0].LAST_NAME as String]" doc:name="Set Payload"  />
```
Comment (marianogonzalez) [inline modules/ROOT/pages/sockets/sockets-connector-examples.adoc]: use `<ee:transform>` instead

File: modules/ROOT/pages/sockets/sockets-connector-examples.adoc [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+	</flow>
+
+	<flow name="socketSendAndReceive">
+		<sockets:listener doc:name="Listener" config-ref="Sockets_Listener_config1" outputMimeType="application/json"/>
+		<sockets:send-and-receive doc:name="Send and receive"  config-ref="Sockets_Request_config" outputMimeType="application/json">
+		</sockets:send-and-receive>
+		<logger level="INFO" doc:name="Logger" message='#[payload]'/>
+		<file:write path="musicians.txt" config-ref="File_Config" mode="CREATE_NEW">
+		  <file:content><![CDATA[#[payload]
```
Comment (marianogonzalez) [inline modules/ROOT/pages/sockets/sockets-connector-examples.adoc]: redundant

File: modules/ROOT/pages/sockets/sockets-connector-examples.adoc [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+		</db:select>
+		<set-payload value="#[output application/json --- 'name': payload[0].FIRST_NAME as String ++ ' ' ++ payload[0].LAST_NAME as String]" doc:name="Set Payload"  />
+	</flow>
+
+	<flow name="socketSendAndReceive">
+		<sockets:listener doc:name="Listener" config-ref="Sockets_Listener_config1" outputMimeType="application/json"/>
+		<sockets:send-and-receive doc:name="Send and receive"  config-ref="Sockets_Request_config" outputMimeType="application/json">
+		</sockets:send-and-receive>
+		<logger level="INFO" doc:name="Logger" message='#[payload]'/>
+		<file:write path="musicians.txt" config-ref="File_Config" mode="CREATE_NEW">
```
Comment (marianogonzalez) [inline modules/ROOT/pages/sockets/sockets-connector-examples.adoc]: you're wrting a json with `txt` extension. People will get confused by this

**Review by marianogonzalez — CHANGES_REQUESTED** [review]

**Review by marianogonzalez — APPROVED** [review]

**Review by kevintroller — APPROVED** [review]
Approved, updated wording - styling.

**Pull request merged**